Stride Formulation (#39)
dwave-examples/portfolio-optimization: README.md

`python portfolio.py -m 'CQM' -t 0.01`

##### Stride Runs

> Similar to the CQM runs, the user can run the same formulations in using the Stride&trade; hybrid solver.

Reviewer: ```suggestion
Similar to the CQM runs, the user can run the same problem using the Stride&trade; hybrid solver.
```

> Similar to the CQM runs the user can run the same formulations in using the latest Stride non-linear solver.

Reviewer: ```suggestion
Similar to the CQM runs, the user can run the same formulations in using the Stride&trade; hybrid solver.
```

```python portfolio.py -m 'Stride'```

> Same parameters as the CQM formulation can be used. The output of the default Stride run is printed on the console as follows.

Reviewer: ```suggestion
The same parameters as the CQM formulation can be used. The output of the default Stride solver run is printed on the console as follows.
```

to do a grid search in order to obtain the best values of the penalty coefficients &gamma;
and the risk-aversion coefficient &alpha; that result in the best objective value.

### CQM and Stride

### Variables
Each x<sub>i</sub> denotes the number of shares of stock i to be purchased.

### Constraints

> All CQM and Stride formulations include the budget constraint described earlier.

Reviewer: ```suggestion
All CQM and Stride solver formulations include the budget constraint described earlier.
```

##### Risk-Bounding Formulation

This includes an upper bound on the risk as a quadratic constraint.

> Such a constraint is supported natively by CQM and Stride.

Reviewer: ```suggestion
Such a constraint is supported natively by the CQM and Stride solvers.
```
